components/director/internal/domain/apptemplate/converter.go

```diff
@@ -81,7 +158,7 @@ func (c *converter) unpackApplicationInput(in string) (*model.ApplicationCreateI
 	return &appInput, nil
 }
 
-func (c *converter) packApplicationInput(in *model.ApplicationCreateInput) (string, error) {
+func (c *converter) packModelApplicationInput(in *model.ApplicationCreateInput) (string, error) {
```

Comment:
ApplicationCreateInputToJSON ? the same for unpack.
Also, if we store applictionCreateInput in string as a JSON, we should mark that it is storing json by appending `JSON` to the field name. The same for Entity.Placeholders